modules/platforms/cpp/thin-client/include/ignite/impl/thin/cache/query/query_fields_cursor_impl.h

```diff
@@ -1,147 +0,0 @@
-/*
```

Comment:
Was this file deleted intentionally?

Reply:
Yeah, it was just some old unused file that was just lying there. Was not used anywhere.